src/test/data/GetInnerRingsCount/MandatoryConstraintThis.ili

```diff
@@ -12,13 +12,19 @@ MODEL TestSuite
 
   TOPIC FunctionTestTopic =
 
+    STRUCTURE ContainerStruct =
+        testAttribute : SURFACE WITH (STRAIGHTS) VERTEX CHKoord WITHOUT OVERLAPS > 0.001;
+    END ContainerStruct;
+
     CLASS BaseClass =
         testAttributeArea : AREA WITH (STRAIGHTS) VERTEX CHKoord WITHOUT OVERLAPS > 0.001;
         testAttributeSurface : SURFACE WITH (STRAIGHTS) VERTEX CHKoord WITHOUT OVERLAPS > 0.001;
         testAttributeMultiSurface : MULTISURFACE WITH (STRAIGHTS) VERTEX CHKoord WITHOUT OVERLAPS > 0.001;
+        testAttrStruct : ContainerStruct;
         MANDATORY CONSTRAINT AreaInnerRingCount:         GeoW_FunctionsExt.GetInnerRingsCount(THIS, "testAttributeArea") == 0;
```

Comment:
PP: Das alignen der Conditions mit Leerzeichen sieht im Moment gut aus. Sollte aber in Zukunft ein weiterer Constraint mit etwas längerem Namen hinzugefügt werden müssten alle anderen Zeilen, die nichts mit dem Change zu Tun haben ebenfalls angepasst werden. Daher würde ich generell code nicht alignen.